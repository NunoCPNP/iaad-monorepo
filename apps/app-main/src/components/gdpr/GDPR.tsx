import { Container } from './GDOR.styles'

const GDPR: React.FC = () => {
  return <Container data-testid="gdpr"></Container>
}

export default GDPR
