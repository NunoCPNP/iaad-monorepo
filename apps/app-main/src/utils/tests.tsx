import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

const AllTheProviders = ({ children }) => {
  return <div>{children}</div>
}

const customRender = (ui, options) => render(ui, { wrapper: AllTheProviders, ...options })

export * from '@testing-library/react'
export { customRender as render }
