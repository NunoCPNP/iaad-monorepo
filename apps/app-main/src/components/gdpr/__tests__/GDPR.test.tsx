import { render, cleanup } from '../../../utils/tests'

import GDPR from '../GDPR'

afterEach(cleanup)

describe('<GDPR /> spec', () => {
  it('Should run without errors', () => {
    const { queryAllByTestId } = render(<GDPR />)

    const gdpr = queryAllByTestId(/gdpr/i)
    expect(gdpr).toHaveLength(1)
  })

  it('Should match snapshot', () => {
    const { asFragment } = render(<GDPR />)

    expect(asFragment(<GDPR />)).toMatchSnapshot()
  })
})
