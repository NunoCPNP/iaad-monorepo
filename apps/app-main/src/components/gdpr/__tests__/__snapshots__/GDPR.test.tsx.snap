// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GDPR /> spec Should match snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="emotion-0"
      data-testid="gdpr"
    />
  </div>
</DocumentFragment>
`;
